lib/screens/v2/candidate_generator/widgets/evergreen.ex

```diff
@@ -5,13 +5,11 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Evergreen do
   alias Screens.V2.WidgetInstance.EvergreenContent
   alias ScreensConfig.Screen
   alias ScreensConfig.V2.EvergreenContentItem
-  alias ScreensConfig.V2.{BusEink, BusShelter, Dup, Elevator, GlEink, PreFare}
 
   def evergreen_content_instances(
-        %Screen{app_params: %app{evergreen_content: evergreen_content}} = config,
+        %Screen{app_params: %_app{evergreen_content: evergreen_content}} = config,
         now \\ DateTime.utc_now()
-      )
-      when app in [BusEink, BusShelter, Dup, Elevator, GlEink, PreFare] do
```

Comment:
This conditional doesn't seem like it was ever necessary right? It would never generate this widget for Sectionals, and `evergreen_content_instances` would only be called for these 6 screen types anyway

Mostly thinking out loud with this comment 😅

Reply:
Correct. It seems like perhaps more of a note/signal for developers that this is the set of screen types we've fully implemented evergreen content on (frontend component mapping and styles). Though IMO it's very unlikely we'd throw a call to generate evergreen widgets into a new screen type and, like, not test at all that it works end-to-end, so this comes across as overly-defensive programming to me.